boxoffice/models/item.py

```diff
@@ -31,6 +31,7 @@ class Item(BaseScopedNameMixin, db.Model):
     assignee_details = db.Column(JsonDict, default={}, nullable=False)
 
     cancellable_until = db.Column(db.DateTime, nullable=True)
+    seq = db.Column(db.Integer, nullable=True)
```

Comment:
`seq` should be non-nullable. SQLAlchemy provides a collection class that you can use to automatically add numbers via `ItemCollection`. See Hascore's NetworkbarLink for how to do this.